.github/workflows/R-CMD-check.yaml

```diff
@@ -32,8 +32,31 @@ jobs:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
+      - uses: n1hility/cancel-previous-runs@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          workflow: R-CMD-check.yaml
+        if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
+
       - uses: actions/checkout@v2
 
+      - name: Set paths
+        if: runner.os == 'Windows'
+        run: |
+          echo "::add-path::C:/rtools40/usr/bin"
+          echo "::add-path::C:/rtools40/mingw64/bin"
+        shell: powershell
```

Comment:
This one and below make sure we use RTools 4.0 toolchain in the tests. That is what we recommend to users and that is what we should test with.